sdcm/remote.py

```diff
@@ -280,9 +299,11 @@ def _run():
                 setattr(result, 'duration', time.time() - start_time)
                 setattr(result, 'exit_status', result.exited)
                 return result
-            except SSHException as ex:
+            except NETWORK_EXCEPTIONS as ex:
                 LOGGER.error(ex)
                 self._ssh_is_up.clear()
+                if is_exception_retriable(str(ex)):
```

Comment:
I assume that on the two other exceptions are different? Because on the other two instances you called is_exception_retriable using ex.result.stderr, while here you just use str(ex)

Reply:
I will rename is_exception_retriable to is_error_retriable. Will that be more clear?
Other two instances run rsync/scp commands using local runner, that has nothing related to SSH. I am just parsing stderr of the rsync/scp commands

Reply:
Yep, that'll be better

Reply:
done